frontend/app/services/DestinationService.scala

```diff
@@ -0,0 +1,50 @@
+package services
+
+import actions._
+import com.netaporter.uri.dsl._
+import configuration.Config
+import model.{ContentDestination, Destination, EventDestination, MembersOnlyContent}
+import play.api.libs.concurrent.Execution.Implicits.defaultContext
+
+import scala.concurrent.Future
+
+trait DestinationService {
+
+  val JoinReferrer = "join-referrer"
```

Comment:
Actually, who knows... looking below, there's all kinds of casing going on. Ignore me!